libs/klv/src/test/java/org/codice/alliance/libs/klv/TestLatitudeLongitudeHandler.java

```diff
@@ -32,6 +35,8 @@
 
     private static final String LON = "lon";
 
+    private static final double EPSILON = 0.01;
```

Comment:
Besides the fact this was only moved to a constant, Is there a reason we're using such a low epsilon? a hundredth of a degree for lat or lon could be something like 1km off